c/src/ml-api-service-offloading.c

```diff
@@ -19,12 +19,12 @@
 #include <string.h>
 #include <curl/curl.h>
 #include <json-glib/json-glib.h>
-#include <nnstreamer-edge.h>
```

Comment:
This .C file has ```nns_edge_connect_type_e``` and ```nns_edge_node_type_e```. nnstreamer-edge.h is still valid in this file.

```
#include "ml-api-service-training-offloading.h"
```
This prevents a compiler error, but it's highly recommended to keep #include in .C file if the intefaces of ```nnstreamer-edge``` are used in this .C file directly. Headers can be kept refactored any time and ```ml-api-service-training-offloading.h``` may remove its dependency on ```nnstreamer-edge``` in the future. This .C file should work regardless of such changes in the header file.

Reply:
I will change this next PR